{{cookiecutter.package_name}}/environment.sh

```diff
@@ -119,21 +131,17 @@ if [[ $create_env == 'yes' ]]; then
     conda remove -n $env_name --all -y
   fi
   # Create conda environment
-  conda create -n $env_name python=3.11 -y
+  conda create -n $env_name python=3.11 -c anaconda -y
   conda activate $env_name
-else
-  echo "Existing environment validated"
-fi
-
-# Install requirements via Github
-if [[ $create_env == 'yes' ]]; then
   # NOTE: update branch name if you update requirements.txt in a branch
   echo "Installing packages for $env_type environment"
   pip install -r $install_file
   # Editable install of repo
   pip install -e .[dev] 
   # Install redis for simulation environments
   if [ $env_type == 'simulation' ]; then
-    conda install redis -y
+    conda install redis -c anaconda -y
```

Comment:
So this is basically _forcing_ users to use anaconda distros, yeah?

Reply:
It will look for packages to install from the anaconda channel.